apps/web/components/eventtype/EventAppsTab.tsx

```diff
@@ -106,9 +108,21 @@ export const EventAppsTab = ({ eventType }: { eventType: EventType }) => {
       {!shouldLockDisableProps("apps").disabled && (
         <div className="bg-muted rounded-md p-8">
           {!isLoading && notInstalledApps?.length ? (
-            <h2 className="text-emphasis text-lg font-semibold">{t("available_apps")}</h2>
+            <>
+              <h2 className="text-emphasis mb-2 text-xl font-semibold leading-5 tracking-[0.01em]">
+                {t("available_apps_lower_case")}
+              </h2>
+              <p className="mb-6 text-sm font-normal text-gray-600">
```

Comment:
you have use color tokens here instead of text-gray-600 because this is how it looks in dark mode

<img width="819" alt="Screenshot 2023-06-19 at 3 08 06 PM" src="https://github.com/calcom/cal.com/assets/53316345/46c813f9-6055-44d9-a9d6-64309eb9a307">

Reply:
i think text-default would work fine here